README.md

```diff
@@ -308,17 +318,60 @@ To fix formatting errors, run the following:
 ```sh
 yarn validate:eslint --fix
 ```
-
 Remember to cover your changes with tests if possible.
 
+# Running the example app 
+While developing, you can run the example app to test your changes.
+
+## Setup
+
+- Clone the repository 
+- Run `yarn` in the root directory to install dependencies, and again in `src` to create the dist build.
+- (on iOS) Run `npx pod-install` from the `example` directory
+- (on Windows) You need to manually link the Slider module to your project.
+
+## Start the app
+
+- Run `yarn run:android` to run on Android
+- Run `yarn run:ios` to run on iOS
+- Run `yarn run:web` to run on web
+
+### Windows
+- Run `yarn run:windows` to run the bundler. Then, run the example app solution in Visual Studio.
+
+# Manual linking of the module on Windows
+On Windows, you need to manually link the `Slider` native module to your project.
+
+#### Add the SliderWindows project to your solution
+
+1. Open the solution in Visual Studio 2019.
+2. Right-click solution icon in Solution Explorer > Add > Existing Project.
+   Select 'D:\pathToYourApp\node_modules\@react-native-community\slider\windows\SliderWindows\SliderWindows.vcxproj'.
+
```

Comment:
isn't this possible with the react-native CLI? most RN devs will be more comfortable with the CLI than VS, could we add instructions to do that?

Reply:
You mean autolinking? I thought it's not available in RNW v62

Reply:
* only sad reacts *